napi-inl.h

```diff
@@ -3661,6 +3661,24 @@ inline const napi_node_version* VersionManagement::GetNodeVersion(Env env) {
   return result;
 }
 
+#if (NAPI_VERSION > 2)
+////////////////////////////////////////////////////////////////////////////////
+// Cleanup class
+////////////////////////////////////////////////////////////////////////////////
+
+inline void Cleanup::AddCleanupHook(Env env, void (*fun)(void* arg), void* arg) {
```

Comment:
No strong feelings, but do we need to repeat the word `Cleanup` here? How about `Napi::Cleanup::AddHook` and `Napi::Cleanup::RemoveHook`?